File: week1/81301/jjunii/81301.py
def solution(s):
    al_num = {"zero": "0", "one": "1", "two": "2", "three": "3", "four": "4", "five": "5", "six": "6", "seven": "7", "eight": "8", "nine": "9"}
    
    for al in al_num:
        if al in s:
            print(al, al_num[al])
            s = s.replace(al, al_num[al])
            
    return int(s)

File: Programmers/week1/81301/kwonjin2/81301.js
function solution(s) {
  const numberWords = [
    "zero",
    "one",
    "two",
    "three",
    "four",
    "five",
    "six",
    "seven",
    "eight",
    "nine",
  ];

  let helper = "";
  let result = "";

  for (let i = 0; i < s.length; i++) {
    if (isNaN(s[i])) {
      helper += s[i];
      if (numberWords.includes(helper)) {
        result += numberWords.indexOf(helper);
        helper = "";
      }
    } else {
      result += s[i];
    }
  }

  return +result;
}
